Ruby/Ruby.sublime-syntax

```diff
@@ -68,7 +68,7 @@ file_extensions:
   - thor
   - Thorfile
   - Vagrantfile
-first_line_match: ^#!/.*\bruby
+first_line_match: ^#!\s*/.*\bruby\b
```

Comment:
Changed this a day or so ago, but the Github UI seems confused...  Maybe the rebase + force push confused it.  The correct version shows in the "Files Changed" tab.